pkg/azurefileplugin/Dockerfile

```diff
@@ -18,7 +18,16 @@ ARG ARCH=amd64
 ARG binary=./_output/${ARCH}/azurefileplugin
 COPY ${binary} /azurefileplugin
 
-RUN apt update && apt upgrade -y && apt-mark unhold libcap2 && clean-install ca-certificates cifs-utils util-linux e2fsprogs mount udev xfsprogs nfs-common netbase
+RUN apt update && apt upgrade -y && apt-mark unhold libcap2 && clean-install ca-certificates cifs-utils util-linux e2fsprogs mount udev xfsprogs nfs-common netbase curl
+
+# install azcopy
+ARG azcopyURL=https://aka.ms/downloadazcopy-v10-linux
+RUN if [ "$ARCH" == "arm64" ] ; then \
+  azcopyURL=${azcopyURL}"-arm64"; fi
+RUN curl -sL ${azcopyURL} | tar -xz && \
```

Comment:
remove curl when azcopy binary is downloaded.

Reply:
`apt remove curl -y`

Reply:
done